<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBox</class>
 <widget class="QGroupBox" name="GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>257</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Master</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="0" column="7">
         <widget class="QComboBox" name="comboBox_7">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Episode</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Project</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="7" colspan="3">
         <widget class="QComboBox" name="comboBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>541</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>541</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QPushButton" name="pushButton_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Preview Master</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Master Shot File Name</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="QRadioButton" name="radioButton">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Custom Shot Template</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QPushButton" name="pushButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Child</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="0" column="7">
         <widget class="QComboBox" name="comboBox_6">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QComboBox" name="comboBox_5">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QComboBox" name="comboBox_4">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label_11">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Version</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
           <string>Build</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Child Shot</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
